### PR TITLE
Proposal for issue 66

### PR DESCRIPTION
**Pull request by goto40**
I added the ability to use search directories.
However, in this case I disabled globbing (see docu: scoping.md).

### CODE REVIEW

File: docs/scoping.md [hunk 1]
```diff
@@ -93,6 +93,15 @@ We provide some standard scope providers:
    Example: see [tests/test_scoping/test_import_module.py](https://github.com/igordejanovic/textX/blob/master/tests/functional/test_scoping/test_import_module.py).
     - `FQNImportURI` (decorated scope provider)
     - `PlainNameImportURI` (decorated scope provider)
+
+   You can use ***globbing*** (import "*.data") with the ImportURI feature.
+   This is implemented via the python "glob" module. Arguments can be passed to
+   the glob.glob function (glob_args), e.g., to enable recursive globbing.
+   Alternatively, you can also specify a list of ***search directories***.
+   In this case no globbing is not allowed and is disabled (reason: it is
```
Comment (igordejanovic) [inline docs/scoping.md]: `no globbing` -> `globbing`

File: tests/functional/test_scoping/issue66/assembly_car1.prog [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import "mylib/assembly.tasks"
+import "mylib/position.tasks"
+skills{
+skill PEG_IN_THE_HOLE_1 {
```
Comment (igordejanovic) [inline tests/functional/test_scoping/issue66/assembly_car1.prog]: Maybe to indent a content of the section for better readability. Also for other model files.

File: textx/scoping/__init__.py [hunk 3]
```diff
@@ -145,7 +146,7 @@ def load_models_using_filepattern(
         Returns:
             the list of loaded models
         """
-        from textx import get_metamodel
+        #from textx import get_metamodel
```
Comment (igordejanovic) [inline textx/scoping/__init__.py]: No need to keep this around.

**Review by igordejanovic — APPROVED** [review]
Looks good. See comments for some minor fixes. I'm approving this so go ahead and merge when fixed.

File: docs/scoping.md (hunk 1 above)
Comment (goto40) [reply docs/scoping.md]: fixed

File: tests/functional/test_scoping/issue66/assembly_car1.prog (hunk 2 above)
Comment (goto40) [reply tests/functional/test_scoping/issue66/assembly_car1.prog]: fixed

File: textx/scoping/__init__.py (hunk 3 above)
Comment (goto40) [reply textx/scoping/__init__.py]: fixed

**Comment by goto40** [comment]
Fixed all issues/comments. All checks passed. I merge now.

**Pull request merged**